domains/headers.go

```diff
@@ -36,8 +36,8 @@ type BlockHeader struct {
 	Bits          uint32         `json:"-"`
 	Nonce         uint32         `json:"nonce"`
 	State         HeaderState    `json:"-"`
-	Chainwork     *big.Int       `json:"-"                 swaggertype:"string"`
-	CumulatedWork *big.Int       `json:"work"              swaggertype:"string"`
+	Chainwork     *big.Int       `json:"-"`
+	CumulatedWork *big.Int       `json:"work"`
```

Comment:
Actually it should be string, we should probably change it on endpoints somehow, or maybe there is a method for Json encoder to have custom converter for big.Int

Reply:
Yes we should return strings from endpoints, so it would be easier to read and will left for consumer the responsibility for parsing it correctly if he has any needs for that field.

Reply:
Done